src/ConnectedVMware/test/Get-AzConnectedVMwareVCenter.Recording.json

```diff
@@ -0,0 +1,44 @@
+{
+  "AzConnectedVMwareVCenter+[NoContext]+Get+$GET+https://management.azure.com/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourceGroups/azcli-test-rg/providers/Microsoft.ConnectedVMwarevSphere/vcenters/azcli-test-vc3?api-version=2022-01-10-preview+1": {
+    "Request": {
+      "Method": "GET",
+      "RequestUri": "https://management.azure.com/subscriptions/204898ee-cd13-4332-b9d4-55ca5c25496d/resourceGroups/azcli-test-rg/providers/Microsoft.ConnectedVMwarevSphere/vcenters/azcli-test-vc3?api-version=2022-01-10-preview",
+      "Content": null,
+      "isContentBase64": false,
+      "Headers": {
+        "x-ms-unique-id": [ "29" ],
+        "x-ms-client-request-id": [ "d1ce93d4-0db0-48d7-a5c3-57de3e4c4144" ],
+        "CommandName": [ "Get-AzConnectedVMwareVCenter" ],
+        "FullCommandName": [ "Get-AzConnectedVMwareVCenter_Get" ],
```

Comment:
Rename this file to AzConnectedVMwareVCenter.Recording.json please

Reply:
done